lib/webpack.js

```diff
@@ -61,7 +61,8 @@ exports.compile = async (outputPath, benchmarkPath, debug) => {
             use: {
               loader: 'babel-loader',
               options: {
-                cwd: projectRoot
+                cwd: projectRoot,
+                presets: ['@babel/preset-env', '@babel/preset-react']
```

Comment:
`babel-loader` should be automatically picking up the target file's Babel config and plugins. This allows users to customise their Babel plugins and settings (like configuring a browser range for `@babel/preset-env`).

Reply:
I'm guessing `.ts` and `.tsx` just need to be added to the resolve extensions and loader regex's.

Reply:
Actually I just realised the problem you're running into is that ui-box no longer has any babel config. If you re-add the babel config and preset dependencies, react-benchmark will work fine.